File: NetFramework/TreeView/CS/BoundModeWithBindingList/Products/Product.cs
using System;

namespace SamplesData
{
    public class Product
    {
        public Guid ID { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }        

        public Product(string name, double price)
        {
            ID = Guid.NewGuid();
            Name = name;
            Price = price;
        }

        /// <summary>
        /// Returns string representation.
        /// </summary>
        public override string ToString()
        {
            return Name;
        }
    }
}

File: NetFramework/TreeView/VB/BoundModeWithBindingList/Products/Product.vb
Public Class Product

    Public Property ID As Guid
    Public Property Name As String
    Public Property Price As Double

    Public Sub New(name As String, price As Double)
        Me.ID = Guid.NewGuid()
        Me.Name = name
        Me.Price = price
    End Sub


    Public Overrides Function ToString() As String
        Return Name
    End Function

End Class
